Scripts/setup_repository.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
+if not "%1"=="1" (
+    powershell -Command "Start-Process -Verb RunAs -FilePath '%0' -ArgumentList '1'"
+    exit /b
+)
+
+cd %~dp0
+
+mkdir "../TF_Application/Assets/TouchFree/"
+mklink /J "../TF_Application/Assets/TouchFree/Tooling" "../TF_Settings_and_Tooling_Unity/Assets/TouchFree/Tooling"
```

Comment:
Tested this .bat on my machine, so I still had some hanging folder meta files in Unity and that meant the .bat didn't copy the files (I assume mkdir errored as there already was one?)

I deleted the folders inside of Unity and then re-ran the .bat and it worked....

I am happy to mark this as known behaviour, but thought it should be noted.

Reply:
I'll throw something for this into the ReadMe just for ease 👍